Packs/Salesforce/Integrations/Salesforce/integration-Salesforce_README.md

```diff
@@ -24,7 +24,7 @@
 <h2>Configure the Salesforce Integration on Demisto</h2>
 <hr>
 <ol>
-<li>Navigate to <strong>Settings</strong> &gt; <strong>Integrations</strong> &gt; <strong>Servers &amp; Services</strong>.</li>
+<li>Navigate to <strong>Settings</strong> &gt; <strong>Integrations</strong> &gt; <strong>Servers `&` Services</strong>.</li>
```

Comment:
are we ok with the `&gt;` here? is it because the entire file is in HTML?

Reply:
yes, run `find   Packs/*/Integrations/ -name '*README.md' | xargs grep '&amp'` from content root. it parses ok in the site.